### PR TITLE
Fix displaying report time in different places

### PR DESCRIPTION
**Pull request by wazsone**
Signed-off-by: Ruslan Bayandinov <wazsone@ya.ru>

# Contribution checklist

## Brief description
Fix few places where time is not rounded properly

image
image

## Checklist

* [ ] - UI test added to added/changed functionality?
* [x] - Screenshot is added to PR if applicable ?
* [x] - Does the code work? Check whether function and logic are correct.
* [x] - Does Changelog.md is updated with changes?
* [x] - Does the translations are up to date?
* [x] - Does it well tested?
* [x] - Tested for Chrome.
* [x] - Tested for Safari.
* [x] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [x] - Rebase your branch onto master and upstream branch
* [x] - Is there any redundant or duplicate code?
* [x] - Are required links are linked to PR?
* [x] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: packages/presentation/src/components/ObjectSearchPopup.svelte [hunk 1]
```diff
@@ -197,15 +197,6 @@
 </form>
 
 <style lang="scss">
-  .overlay {
```
Comment (BykhovDenis) [inline packages/presentation/src/components/ObjectSearchPopup.svelte:200]: Why?

**Review by BykhovDenis — COMMENTED** [review]
Use `rush format` for formatting

File: packages/presentation/src/components/ObjectSearchPopup.svelte (hunk 1 above)
Comment (wazsone) [reply packages/presentation/src/components/ObjectSearchPopup.svelte:200]: @BykhovDenis It is not used

File: plugins/tracker-resources/src/components/sprints/SprintEditor.svelte [hunk 2]
```diff
@@ -64,47 +64,53 @@
...
+              if (cReported < cEstimation) {
+                retEst = cReported
+              }
+            }
+          }
+        } else {
           if (cat === tracker.issueStatusCategory.Completed || cat === tracker.issueStatusCategory.Canceled) {
-            if (cReported < cEstimation) {
-              retEst = cReported
+            if (it.reportedTime < it.estimation) {
```
Comment (haiodo) [inline plugins/tracker-resources/src/components/sprints/SprintEditor.svelte:86]: Please revert to (cReported < cEstimation)

Comment (wazsone) [reply plugins/tracker-resources/src/components/sprints/SprintEditor.svelte:86]: @haiodo I did not change that, I only put expression into ```floorFractionDigits```.

It is git displaying that way:

image

**Review by BykhovDenis — APPROVED** [review]

**Pull request merged**